scripts/Units/legmohoconin.bos

```diff
@@ -0,0 +1,58 @@
+
+#include "../recoil_common_includes.h"
+
+piece  base, aim, nano1, nano2, rotor, attach;
```

Comment:
this is the hidden one right?, why does it have so many pieces?

Reply:
Same .s3o file as the others, and it has to hide all its parts. I figured this would be cheaper than having a seperate single piece model, since it needs to be the same model to give the correct radar ghost.